frontend/src/components/auth/LoginForm.svelte

```diff
@@ -1,41 +1,49 @@
 <script>
-    export let registerToggle
+    import { auth } from '../../stores/auth.js'
 
-    // 아이디 패스워드 감지
-    let memberId = ''
-    $: memberId
+    export let registerToggle
 
-    let password = ''
-    $: password
+    const data = {
+        memberId: '',
+        password: ''
+    }
 
     // 로그인 버튼 제어
     let isSubmitLock = true
-    $: isSubmitLock = memberId.trim() === '' || password.trim() === ''
+    $: isSubmitLock = data.memberId.trim() === '' || data.password.trim() === ''
 
     function onRegisterMode() {
         registerToggle = true
     }
+
+    const resetData = () => {
+        data.memberId = ''
+        data.password = ''
+    }
+
+    const onLogin  = async () => {
+        await auth.login(data)
+        resetData()
```

Comment:
로그인에 성공할 때만 resetData에서 전체를 초기화하고, 실패하면 패스워드 부분만 초기화하는게 어때요?
```suggestion
    const const resetData = (isOk) => {
        if (isOk) {
            data.memberId = '';
            data.password = '';
        } else {
            data.password = '';
        }
    }

    const onLogin  = async () => {
        const isOk = await auth.login(data)
        resetData(isOk)
```

그런데 지금보니, auth.login 부분에는 try-catch가 없는데, 나중에 수정하면 좋을거같아요